src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -545,6 +545,54 @@ async Task<ArraySegment<byte>> WriteLiteralExpressionAsIndex(DotnetObjectId root
             }
         }
 
+        private static bool CheckParametersCompatibility(ElementType? typeCode, JObject value)
+        {
```

Comment:
Don't we already do this kind of parameter/argument type compatibility check for regular method calls? Can that be used here?

Reply:
No, we don't do, we only believe that the customer is calling with the correct parameters and try to run it on runtime side, if we have a type mismatch the runtime will return an error and we return the error to the ide.